src/script/lua_api/l_noise.h

```diff
@@ -150,7 +151,8 @@ class LuaPcgRandom : public ModApiBase
 
 public:
 	LuaPcgRandom(u64 seed) : m_rnd(seed) {}
-	LuaPcgRandom(u64 seed, u64 seq) : m_rnd(seed, seq) {}
+	// @TODO: set seq
+	LuaPcgRandom(u64 seed, u64 seq) : m_rnd(seed) {}
```

Comment:
i should verify that case and ensure we can have same values, random with same seed and same uniform algorithm should match